spec/models/post.spec.js

```diff
@@ -21,6 +21,11 @@ describe("Post model", () => {
     expect(post.date).toEqual( Date().slice(0, -31) );
   });
 
```

Comment:
Do we need extra unit tests for like function? We now have like = 0 when initialising, maybe we also need to show the like number increase too.